dockers/docker-database/supervisord.conf.j2

```diff
@@ -31,6 +31,8 @@ dependent_startup=true
 {% if INSTANCES %}
 {%     for redis_inst, redis_items in INSTANCES.items() %}
 {%- if redis_inst != 'remote_redis' %}
+
```

Comment:
Suggest remove unnecessary change in this file

Reply:
https://github.com/sonic-net/sonic-buildimage/pull/20545

Reply:
#Closed